.github/workflows/check-compatibility.yml

```diff
@@ -11,13 +11,15 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: Run compatibility task
-        run: ./gradlew checkCompatibility | tee $HOME/gradlew-check.out
+        run: ./gradlew checkCompatibility -i | tee $HOME/gradlew-check.out
 
       - name: Get results
         run: |
-          echo 'Compatibility status:' > ${{ github.workspace }}/results.txt && echo '```' >> ${{ github.workspace }}/results.txt
-          grep -e 'Compatible components' -e 'Incompatible components' -e 'Components skipped' -A 2 -B 3 $HOME/gradlew-check.out >> "${{ github.workspace }}/results.txt"
-          echo '```' >> ${{ github.workspace }}/results.txt
+          echo '## Compatibility status:' > "${{ github.workspace }}/results.txt"
```

Comment:
@Yury-Fridlyand What do you think? Ran this on my local machine, it would could like this:
> ## Compatibility status:
>Checks if related components are compatible with change fe1e87eabf4
>### Incompatible components
>- https://github.com/opensearch-project/sql.git
>### Skipped components
>- https://github.com/opensearch-project/skippie.git
>### Compatible components
>- https://github.com/opensearch-project/security.git
>- https://github.com/opensearch-project/foo.git

Reply:
Resolving this comment, I've updated the description to include this example